mlbaklava/resources/deploy/main.py

```diff
@@ -130,13 +131,20 @@ def invocations():
                           default: 1
         """
 
-        if request.content_type == 'application/json':
+        if request.is_json:
             data = request.get_json()  # type: dict[str, object]
         else:
             data = request.get_data()  # type: bytes
 
+        # Create prediction adapter
+        da = DeployAdapter(os.getenv('sriracha_provider'))
+
         # Execute lambda
-        result = func(data)
+        try:
```

Comment:
Not sure, this would work. 

There are scenarios where original entry function has multiple argument. Such scenarios will fail and errors will be masked. 

We need to figure out better way to make Job adapter optional